modules/foreman_setting.py

```diff
@@ -81,14 +81,17 @@
     name: "http_proxy"
 '''
 
-RETURN = ''' # '''
+RETURN = '''
+entity:
```

Comment:
I'm not sure calling this `entity` is user-friendly, any other suggestions?

Reply:
could be `setting` but maybe `foreman_setting` is better in avoiding confusion.

Reply:
either works for me :+1: